project/CLI.py

```diff
@@ -30,26 +30,27 @@ def __login(self, args):
         return "Login failed"
 
     @need_admin
-    def __add_team(self, args):
+    def __addteam(self, args):
```

Comment:
it is more "pythonic" to use the underscore to separate words in a method. Pycharm is probably warning you about the new name